apps/sync/js/adapters/bookmarks.js

```diff
@@ -65,16 +65,18 @@ var BookmarksHelper = (() => {
     Due to this common requirement, bug 1207468 will have an improvement for
     this case.
   */
-  function setDataStoreId(synctoId, dataStoreId) {
+  function setDataStoreId(synctoId, dataStoreId, userid) {
```

Comment:
Actually, improving comments in general on adapters code would be really helpful.